diff --git a/peekaboo_dr/CMakeLists.txt b/peekaboo_dr/CMakeLists.txt
index 2da9acf..abba365 100755
--- a/peekaboo_dr/CMakeLists.txt
+++ b/peekaboo_dr/CMakeLists.txt
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.5.1)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+set(CMAKE_C_STANDARD 11)
+set(CMAKE_C_STANDARD_REQUIRED ON)
 set(OPT_CFLAGS "${OPT_CFLAGS} -DDEBUG")
 find_package(DynamoRIO)
 if (NOT DynamoRIO_FOUND)
diff --git a/peekaboo_dr/peekaboo_dr.c b/peekaboo_dr/peekaboo_dr.c
index 26dd52e..6c7f295 100755
--- a/peekaboo_dr/peekaboo_dr.c
+++ b/peekaboo_dr/peekaboo_dr.c
@@ -328,7 +328,7 @@ static void instrument_insn(void *drcontext, instrlist_t *ilist, instr_t *where,
 }
 
 
-static dr_emit_flags_t save_bb_rawbytes(void *drcontext, void *tag, instrlist_t *bb, bool for_trace, bool translating, void **user_data)
+static dr_emit_flags_t save_bb_rawbytes(void *drcontext, void *tag, instrlist_t *bb, char for_trace, char translating, void **user_data)
 {
 	per_thread_t *data = drmgr_get_tls_field(drcontext, tls_idx);
 	bytes_map_t bytes_map[MAX_NUM_BYTES_MAP];
@@ -357,7 +357,7 @@ static dr_emit_flags_t save_bb_rawbytes(void *drcontext, void *tag, instrlist_t
 }
 
 static dr_emit_flags_t per_insn_instrument(void *drcontext, void *tag, instrlist_t *bb, instr_t *instr, 
-		                             bool for_trace, bool translating, void *user_data)
+		                             char for_trace, char translating, void *user_data)
 {
 	drmgr_disable_auto_predication(drcontext, bb);
 	if (!instr_is_app(instr)) return DR_EMIT_DEFAULT;
